spinalcordtoolbox/gui/widgets.py

```diff
@@ -63,7 +65,7 @@ def refresh(self, labels=None):
             self._checked = labels
             self._unchecked = set(self._check_boxes.keys()) - set(labels)
 
-        for checkbox in self._check_boxes.itervalues():
+        for checkbox in self._check_boxes.values():
```

Comment:
produces the same result? what's the difference with itervalues?

Reply:
In python3, `dict.values()` is always returning an iterator, whereas in Python2 `dict.values()` is creating a list. There's no practical difference in this case.